pkg/ui/src/components/NoteButton/index.ts

```diff
@@ -0,0 +1,3 @@
+import NoteButton from "./NoteButton";
+
+export default NoteButton;
```

Comment:
You can also add `export * from "./NoteButton"` here as you want the consumer of `NoteButton` (in web package perhaps) be able to use the above specified enums (NoteButtonType, etc.) and it's always ok to export the Props interface for a component.